automation/project/Dependencies.scala

```diff
@@ -35,13 +37,14 @@ object Dependencies {
     "com.typesafe.akka"   %%  "akka-testkit"        % akkaV     % "test",
     "com.typesafe.akka"   %%  "akka-slf4j"          % akkaV,
     "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
-    "org.scalatest"       %%  "scalatest"     % "2.2.6"   % "test",
+    "org.scalatest"       %%  "scalatest"     % "3.0.1"   % "test",
```

Comment:
This update to match `workbench-service-test` required changes in how we use `eventually`